kernel/kthread.c

```diff
@@ -319,6 +319,7 @@ struct task_struct *kthread_create_on_node(int (*threadfn)(void *data),
 		 */
 		sched_setscheduler_nocheck(task, SCHED_NORMAL, &param);
 		set_cpus_allowed_ptr(task, cpu_all_mask);
```

Comment:
Why are both of these lines still here? Isn't there one line where they can be scheduled anywhere?